src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Write.Stream.cs

```diff
@@ -19,7 +18,7 @@ public static partial class JsonSerializer
         // We check for flush after each JSON property and element is written to the buffer.
         // Once the buffer is expanded to contain the largest single element\property, a 90% thresold
         // means the buffer may be expanded a maximum of 4 times: 1-(1/(2^4))==.9375.
-        private const float FlushThreshold = .90f;
+        internal const float FlushThreshold = .90f;
```

Comment:
I think you should move it to a new file - it doesn't seem to be used anywhere here

Reply:
I figured I'd keep it here so that we can have git history on when/why it was added.